analyzers/src/SonarAnalyzer.ShimLayer/Sonar/IOperationWrapper.cs

```diff
@@ -0,0 +1,44 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2024 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace StyleCop.Analyzers.Lightup;
+
+public interface IOperationWrapper
```

Comment:
We need to make a decision here: this file was present in an old version of StyleCop, removed after [this commit](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/commit/9b5db190f53bc6e8689b4df433684d92868fb6b0#diff-bc332288d73c300d401c5f80288acfb80ed57377f721cc9e141f804d0c01c442) by generating it through the AbstractNode defined in `OperationInterfaces.xml`. This node is ad hoc, specific for stylecop, and not present either in Roslyn or on our XML. We do have also an `IOperationWrapperSonar.cs` that was a temporary substitute.

Reply:
I would keep it like this for now since we import the file from Roslyn, and the XML element used to generate it is only part of StyleCop and missing from Roslyn.

Reply:
I would like to avoid importing one part from Roslyn and another one from StyleCop.